concrete/src/Html/Object/Picture.php

```diff
@@ -31,39 +31,73 @@ class Picture extends Element
     //////////////////////////// CORE METHODS //////////////////////////
     ////////////////////////////////////////////////////////////////////
 
-    public function __construct(array $sources = array(), $fallbackSrc, $attributes = array())
+    public function __construct(array $sources = array(), $fallbackSrc, $attributes = array(), $lazyLoadNative, $lazyLoadJavaScript)
```

Comment:
The new parameters should have defaults (false? or null?), so preserving compatibility with other uses that may not have implemented the new parameters.

Reply:
Oops, see you have already done that for create